addons/sourcemod/scripting/WCX_Engine_Crit.sp

```diff
@@ -8,30 +8,30 @@ public Plugin:myinfo =
     description="Generic crit skill"
 };
 
-public OnWar3EventPostHurt(victim, attacker, Float:damage, const String:weapon[32], bool:isWarcraft)
+public int OnWar3EventPostHurt(int victim, int attacker, float damage, const char weapon[32], bool isWarcraft)
```

Comment:
SourcePawn always had a (implicit) integer return value(prior to new syntax) which simply returned zero.
If I declare _OnWar3EventPostHurt_ as void the compiler complains about a prototype mismatch(because the forward is in old syntax -> int return value). Easy fix is to set return value as integer and return 0 to replicate old behavior.